atlp-rwanda/elite-bn-be
# ft(user-login): user login to Barefoot via email & password

#### What does this PR do?
Registered user can login to Barefoot using email and password

#### How should this be manually tested?

 After cloning this PR follow those steps to test it 
 1. Create `.env` file and follow the information in `.env.example` file
 2. Make sure you choose the environment you want to work with NODE_ENV
 3. Then use this `localhost:3000/api-docs` to use swagger for testing this PR
 4. And then use this `POST:`  `api/v1/user/login` for postman
 5.  Use email and password as body
 
#### Any background context you want to provide?
Run `npm run migration` to have a database table 
Run  `npm start` to start the server 

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/182201569
